README.md

```diff
@@ -1,5 +1,7 @@
 # docker-sonar-scanner-node
 
+**This repo has been deprecated in favour of the [docker-sonar-scanner repo](https://github.com/UKHomeOffice/docker-sonar-scanne). If you require any extras packages, Feel free to fork the repo**
```

Comment:
The URL is missing an `r` and needs correcting

Reply:
Thanks for pointing that out. I've added the `r`